### PR TITLE
Add dependencies for macOS

### PR DESCRIPTION
**Pull request by alper**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,17 @@ Whenever you want to update:
...
 ```
 
+Run `cargo mobile doctor` to check your installation.
+
+### macOS
+
+On macOS you're going to need:
+
+```
+brew install xcodegen
```
Comment (lucasfernog) [inline README.md:44]: We install these automatically: https://github.com/tauri-apps/tauri-mobile/blob/6004ce22371943d7b85992e0348155a2f9e95f04/src/apple/deps/mod.rs#L189

Comment (alper) [reply README.md:44]: I was going to add `libimobiledevice` which I just ran into.

But then how come I didn't get them?

Comment (lucasfernog) [reply README.md:44]: Are you using tauri-mobile or the Tauri CLI?

Comment (alper) [reply README.md:44]: I got here following this: https://github.com/tauri-apps/tauri-mobile#installation

**Pull request closed without merging**